index.js

```diff
@@ -1,8 +1,39 @@
 "use strict";
 var window = require("global/window")
-var isFunction = require("is-function")
-var parseHeaders = require("parse-headers")
-var xtend = require("xtend")
+var _extends = require("@babel/runtime/helpers/extends");
```

Comment:
Do we run XHR through babel? I'm not sure this would get deduped properly.

Reply:
We run it through rollup, and rollup will dedupe it for us

Reply:
babel just adds in the `import`/`require` statement for us using `@babel/plugin-runtime-transform` so we will have a duplicate dependency and rollup will see that and dedupe them.